### PR TITLE
Add compose option to createStore

### LINKED ISSUE #65: Pass options to devtools
Although it is convenient to include devtools integration in this library, there's a fair question to ask here: is it this library's responsibility to do it? If it's done for technical reasons, I think we need a way to pass options to the devtools' compose function.

Particularly action blacklisting, state sanitization, max-age are invaluable for large apps.

In addition, different apps have different needs (e.g. using devtools in production etc.). There are infinitely many possible ways to configure devtools. What if `createStore` took a `compose` function as a parameter?

Another alternative (as seen in other issues/questions submitted) is to have an additional API that exports enhancers/middlewares for dynamic module management and let users seeking more freedom use `createStore` from `redux`.

If you would like to go with the first option (adding compose param to `createStore`), I can make a PR. I suggest that `createStore`’s first argument is made an options object in that case e.g. `createStore<...>(options: { compose, initialState, enchancersEtc }, ...modules)`. I’m typing from mobile so this is not a type signature :smile:

It seems it’d be better if this were done after https://github.com/Microsoft/redux-dynamic-modules/pull/55 is merged. Nevertheless this will be a breaking change.

### PR DESCRIPTION
**Pull request by anilanar**
Fixes #65

### CODE REVIEW

**Comment by anilanar** [comment]
@abettadapur Can I somehow help for this to get merged?

File: packages/redux-dynamic-modules-core/src/ModuleStore.ts [hunk 1]
```diff
@@ -17,6 +18,7 @@ type ModuleStoreSettings<S> = {
     initialState?: DeepPartial<S>;
     enhancers?: StoreEnhancer[];
     extensions?: IExtension[];
+    compose?: typeof compose;
```
Comment (abettadapur) [inline packages/redux-dynamic-modules-core/src/ModuleStore.ts]: nit, call advancedComposeEnhancers, just as the combine function below

**Comment by abettadapur** [comment]
@anilanar Made one tiny comment, just rename the option, and I'll merge

**Comment by anilanar** [comment]
@abettadapur Done.

**Comment by codeocelot** [comment]
@anilanar looks like you need to rebase your changes.  Personally I'm stoked for this long awaited feature.

**Comment by anilanar** [comment]
@abettadapur Rebased.

**Pull request merged**